package.json

```diff
@@ -6,10 +6,18 @@
     "url": "https://github.com/datawrapper/datawrapper.git"
   },
   "scripts": {
+    "format": "prettier --write 'src/**/*.{js,html}'",
+    "lint": "healthier 'src/**/*.{js,html}' && prettier --check 'src/**/*.{js,html}'",
```

Comment:
Is `healthier` not running the `prettier` check? And maybe the format script could be `healthier --write 'src/**/*.{js,html}'`. I don't know if that's how it works 😅